### ISSUE TITLE
`make check` doesn't support FreeBSD

### ISSUE THREAD

**Issue by slackner** [Winetricks/winetricks#492]
As mentioned [here](https://github.com/Winetricks/winetricks/pull/488#issuecomment-87366724) the `tests/winetricks-test` script fails on FreeBSD with:

```
time: illegal option -- -
usage: time [-al] [-h | -p] [-o file] utility [argument ...]
sort -k -k3.1: Invalid argument
...
```

All tests are counted as failures, although they work fine when executed manually directly with winetricks.

**Comment by austin987** [comment]
First issue is fixed by
https://github.com/Winetricks/winetricks/commit/420c8ab5b03f46e8d1561a7bd7a16a9b2c2d010e

the sort issue remains, I'll take a look when I get access to BSD.

**Comment by austin987** [comment]
Sort issue is fixed by 20ad68840fbd56f72ae9dfa16401c8472fb9dff8
